eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -188,15 +187,12 @@ private static void run(final List<String> opts) throws Exception {
             );
         }
         for (int idx = 1; idx < opts.size(); ++idx) {
-            final Phi phi = new EOstring(Phi.Φ);
-            final String arg = opts.get(idx);
-            phi.attr("Δ").put(new Data.Value<>(arg));
-            app.attr(0).put(phi);
+            app.attr(0).put(new Data.ToPhi(opts.get(idx)));
         }
         Main.LOGGER.info(
             String.format(
                 "%n---%n%s",
-                new Dataized(app).take().toString()
+                Arrays.toString(new Dataized(app).take())
```

Comment:
@c71n93 let's add a puzzle here so we won't forget to make up with some type casting here

Reply:
@maxonfjvipon done. Could you check, please?